share/picongpu/examples/LaserWakefield/lib/python/picongpu/params.py

```diff
@@ -18,10 +18,13 @@
 
 from picongpu.input.parameters import LogScaledParameter, LinearScaledParameter
 
+pico = 1.e-12
+dt_r = 1. / 1.39e-16 * pico
```

Comment:
REQUIRED:
Please explain this magic number (`1.39e-16`) in a comment or give it a name that makes it clear (as e.g. pico).

Reply:
this is the inverse (historically suffixed `_r`) of `dt`.

We might be able to just add is as a `Parameter` (but not `UiParameter`) in the list below to make it more accessible.